lib/hobo_fields/model.rb

```diff
@@ -234,7 +240,7 @@ def add_index_for_field(name, args, options)
         if (to_name = options.delete(:index))
           index_opts =
             {
-              unique: args.include(:unique) || options.delete(:unique)
+              unique: args.include?(:unique) || options.delete(:unique)
```

Comment:
Whoa, how did this code work before (with no `?` on the end of `include`)?

Reply:
It didn't, code/tests just hadn't touched it because of either not being run or being broken